infra/gcp/terraform/k8s-infra-prow-build-trusted/secrets.tf

```diff
@@ -104,3 +108,22 @@ resource "google_secret_manager_secret_iam_member" "k8s_prow_kes_sa_role_accesso
   role      = "roles/secretmanager.secretAccessor"
   member    = "serviceAccount:kubernetes-external-secrets-sa@k8s-prow.iam.gserviceaccount.com"
 }
+
```

Comment:
Can you delete these last 3 objects? It is not needed. The rest of the PR looks good

Reply:
+1 let's try without these

Reply:
Done.